content/docs/conditional-rendering.md

```diff
@@ -143,11 +143,15 @@ root.render(<Mailbox unreadMessages={messages} />);
 
 [**Try it on CodePen**](https://codepen.io/gaearon/pen/ozJddz?editors=0010)
 
-It works because in JavaScript, `true && expression` always evaluates to `expression`, and `false && expression` always evaluates to `false`.
+The `&&` operator returns the left operand if it is falsy, otherwise if the left operand is truthy it returns the right operand.
 
-Therefore, if the condition is `true`, the element right after `&&` will appear in the output. If it is `false`, React will ignore and skip it.
+React will render any value that is not `false`.
```

Comment:
Need to fix this to say exactly for which which falsy values it doesn’t render, like `null` and `undefined`.